KutokAccounting.DataProvider/Models/TransactionType.cs

```diff
@@ -3,7 +3,7 @@ namespace KutokAccounting.DataProvider.Models;
 public class TransactionType
 {
     public int Id { get; set; }
-    public string? Name { get; set; }
+    public string Name { get; set; } = null!;
```

Comment:
```suggestion
    public required string Name { get; set; }
```
Не треба супресити null, потім тобі ж боком вилізе